### ISSUE TITLE
Pages always marked as index,follow

### ISSUE THREAD

**Issue by mallardduck** [tuxnull/BootstraPHP#9]
**Describe the bug**
Within the `init_meta` function you set a static value for the robots meta tag. Found here: 
https://github.com/tuxnull/BootstraPHP/blob/5aa1cbec96060c4cbcee9595cb4c206b9effe5fc/bootstrap.php#L9

From an SEO perspective, it's less than ideal that this package sets all pages to `index,follow` via a robots meta tag. This is a meta tag that should be used with caution and is generally adjusted on a page/URL by page/URL basis.

For example, some pages may necessitate: `index, nofollow`, `noindex, follow` or `noindex, nofollow`. Any premutation of these could exist 

**To Reproduce**
See: https://github.com/tuxnull/BootstraPHP/blob/5aa1cbec96060c4cbcee9595cb4c206b9effe5fc/bootstrap.php#L9

**Expected behavior**
A user should be able to control the values this tag provides.

**Comment by tuxnull** [comment]
You're right. Will remove it until I find a good way of making it more user controllable.
